src/ahbichtfunctionsclient/__init__.py

```diff
@@ -1,5 +1,6 @@
 """
-Contains an asynchronous Package Resolver that, by default, uses the Hochfrequenz API to resolve packages.
+Contains two asynchronous Resolver that, by default, uses the Hochfrequenz API to request a
+Condition Expression or a Condition Text for a given Package/Condition Key.
```

Comment:
```suggestion
condition expression or a condition text for a given package/condition key.
```

eigennamen wie `ConditionKeyConditionTextMapping` groß, sonst aber expression und package und condition als lower case.